infrastructure/deployment/deploy.sh

```diff
@@ -64,7 +64,7 @@ done
 
 # Default values
 SSH_PORT=22
-SSH_ARGS=${SSH_ARGS:-""}
+SSH_ARGS=${SSH_ARGS:-}
```

Comment:
These types of changes are related to
https://github.com/opencrvs/opencrvs-farajaland/pull/953